pkg/pmem-csi-operator/controller/deployment/testcases/testcases.go

```diff
@@ -69,7 +69,7 @@ func UpdateTests() []UpdateTest {
 			}
 		},
 		"pmemPercentage": func(d *api.Deployment) {
-			d.Spec.PMEMPercentage++
+			d.Spec.PMEMPercentage--
```

Comment:
When does that test failure occur? We don't see it when testing "devel" and it is not obvious why the other commits should cause it.

Reply:
Which E2E test fails?

Reply:
https://cloudnative-k8sci.southcentralus.cloudapp.azure.com/blue/organizations/jenkins/pmem-csi/detail/PR-718/11/tests/

Reply:
I don't like unsolved mysteries like "test randomly fails". Let's take this fix out of this PR and discuss it separately.